### PR TITLE
builder/packages: add dependencies for building android

### PR DESCRIPTION
**Pull request by craftyguy**

### CODE REVIEW

**Comment by craftyguy** [comment]
I know some of these are most likely already pulled into the install, but I feel that listing them explicitly can't hurt, right?

**Review by dcbaker — CHANGES_REQUESTED** [review]
A bunch of these are duplicated in the slave config (which is applied to builders)

among them:
bc
bison
build-essential
ccache
libc*
python-mako

Also, why is distcc and gpg required

**Comment by craftyguy** [comment]
@dcbaker changes made, and also added the repo script to salt as we discussed yesterday!

File: builder/packages.sls [hunk 1]
```diff
@@ -26,3 +26,34 @@ builder-packages:
           - libllvm4.0
           - libllvm4.0:i386
           - llvm-4.0-dev
+          - curl
+            # Android lists this as a dependency
+          - distcc
+          - dosfstools
+          - gettext
+          - git-core
```
Comment (dcbaker) [inline builder/packages.sls:34]: I think this will be pulled in with git from slave

File: builder/packages.sls [hunk 2]
```diff
@@ -26,3 +26,34 @@ builder-packages:
...
+          - libc6-dev-i386
+          - libgl1--dev
+          - libncurses5-dev:i386
+          - libreadline6-dev:i386
+          - libxml2-utils
+          - lunch
+          - mtools
+          - openjdk-8-jdk
+          - python2.7
+          - python-mako
```
Comment (dcbaker) [inline builder/packages.sls]: python-mako is in salve, and python2.7 will be pulled in implicitly. I'd rather add this to slave if we're going to add it

**Review by dcbaker — CHANGES_REQUESTED** [review]

File: builder/packages.sls (hunk 1 above)
Comment (craftyguy) [reply builder/packages.sls:34]: I don't think `git` pulls in `git-core`, at least the debian package page for git doesn't list git-core as a dependency: https://packages.debian.org/sid/git

Comment (dcbaker) [reply builder/packages.sls:34]: Okay, ignore this comment then :)

**Review by dcbaker — APPROVED** [review]

**Comment by janesma** [comment]
pushed

**Pull request closed without merging**